### PR TITLE
Remove dependencies on environment variables

### PR DESCRIPTION
**Pull request by pixelpogo**
In general it is cumbersome for the users of a Gem to have to deal with environment variables, which are specific only to the Gem itself. It is easier to hand-in all relevant information with a configure block.

This PR removes the dependency on the dot.env gem and removes the `.env*` files and makes all options configurable with `Dynameister.configure`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +41,7 @@ This is how Dynameister can be configured, e.g. in an initializer in a Rails app
     config.write_capacity 350
     # config.endpoint "http://192.168.99.100:32768"
     config.region "eu-west-1"
+    config.credentials Aws::Credentials.new( "access_key_id", "secret_access_key","session_token")
```
Comment (baschtl) [inline README.md]: Whitespace missing at the last parameter.

File: spec/dynameister_spec.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
 describe Dynameister do
 
-  let(:capacity) { 99 }
-  let(:endpoint) { 'foo.bar' }
-  let(:region)   { 'dyna-west-1' }
+  let(:capacity)    { 99 }
+  let(:endpoint)    { 'foo.bar' }
+  let(:region)      { 'dyna-west-1' }
+  let(:credentials) { Aws::Credentials.new( "access_key_id", "secret_access_key","session_token") }
```
Comment (baschtl) [inline spec/dynameister_spec.rb]: See https://github.com/babbel/dynameister/pull/11/files#r53140087.

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -3,18 +3,17 @@
...
 
-# Load environment-specific settings
-ENV['DYNAMEISTER_ENV'] ||= "test"
-Dotenv.load(File.join(File.dirname(__FILE__), ".env.#{ENV['DYNAMEISTER_ENV']}"))
-
-Dynameister.endpoint ENV['DYNAMEISTER_ENDPOINT']
+Dynameister.configure do |config|
+  config.endpoint (ENV['DYNAMEISTER_ENDPOINT'] || "http://localhost:8000")
+  config.region "dynameister-test"
+  config.credentials Aws::Credentials.new( "access_key_id", "secret_access_key","session_token")
```
Comment (baschtl) [inline spec/spec_helper.rb]: See https://github.com/babbel/dynameister/pull/11/files#r53140087.

File: README.md [hunk 4]
```diff
@@ -254,9 +250,14 @@ To avoid too many complaints of HoundCI in your pull request use a [plugin for y
...
 ## Testing
 
-1. Copy `spec/.env.test.template` to `spec/.env.test`,
-2. Adapt `spec/.env.test` according to its comments,
-3. Run `bundle exec rspec`.
+In order to run the rspec test suite please enter
+
+`bundle exec rspec`
+
+Dynameister expects *DynamoDBLocal* to be running on localhost on port 8000. If you’re local environment is different, then you have to specify `ENV['DYNAMEISTER_ENDPOINT‘]`.
```
Comment (baschtl) [inline README.md]: `If your local...`

**Comment by baschtl** [comment]
Looks good to me :+1:

File: spec/spec_helper.rb [hunk 5]
```diff
@@ -3,18 +3,17 @@
...
 
 require "dynameister"
 
-# Load environment-specific settings
-ENV['DYNAMEISTER_ENV'] ||= "test"
-Dotenv.load(File.join(File.dirname(__FILE__), ".env.#{ENV['DYNAMEISTER_ENV']}"))
-
-Dynameister.endpoint ENV['DYNAMEISTER_ENDPOINT']
+Dynameister.configure do |config|
+  config.endpoint (ENV['DYNAMEISTER_ENDPOINT'] || "http://localhost:8000")
```
Comment (nicsnet) [inline spec/spec_helper.rb]: Double quotes are preferred.

**Comment by nicsnet** [comment]
+B great addition!

**Comment by pixelpogo** [comment]
@baschtl and @nicsnet Thank you for your feedback! :-)

**Pull request merged**